Microsoft.Azure.Cosmos.Samples/Tools/CTL/Scenarios/ChangeFeedPullScenario.cs

```diff
@@ -0,0 +1,183 @@
+            }
+
+            if (config.PreCreatedDocuments > 0)
+            {
+                logger.LogInformation("Pre-populating {0} documents", config.PreCreatedDocuments);
+                await Utils.PopulateDocumentsAsync(config, logger, new List<Container>() { cosmosClient.GetContainer(config.Database, config.Collection) });
+            }
+        }
+
+        public async Task RunAsync(
+            CTLConfig config,
+            CosmosClient cosmosClient,
+            ILogger logger,
+            IMetrics metrics,
+            string loggingContextIdentifier,
+            CancellationToken cancellationToken)
+        {
+            Stopwatch stopWatch = new Stopwatch();
+            stopWatch.Start();
+
+            //Repeat the loop of creating a new Change Feed Iterator and all the steps until the run-time is completed.
+            while(stopWatch.Elapsed <= config.RunningTimeDurationAsTimespan)
+            {
+                int documentTotal = 0;
+                string continuation = null;
+                Container container = cosmosClient.GetContainer(config.Database, config.Collection);
+                FeedIterator<Dictionary<string, string>> changeFeedPull = container.GetChangeFeedIterator<Dictionary<string, string>>
+                                                                                (ChangeFeedStartFrom.Beginning(), ChangeFeedMode.Incremental);
+                try 
+                {
+                    while (changeFeedPull.HasMoreResults)
+                    {
+                        //Call ReadNextAsync on the iterator
+                        FeedResponse<Dictionary<string, string>> response = await changeFeedPull.ReadNextAsync();
+                        //If there are new documents in the response, count them in a variable
+                        documentTotal += response.Count;
+                        continuation = response.ContinuationToken;
+                        //If there are no new changes, stop/break out of the iterator.
+                        if (response.StatusCode == HttpStatusCode.NotModified)
+                        {
+                            break;
+                        }                   
+                    }
+                    if(config.PreCreatedDocuments == documentTotal) 
+                    {
+                        logger.LogInformation("Suceess: The number of new documents match the number of pre-created documents");
+                    }
+                    //Compare the detected changes vs the prepopulated ones and if they don't match, log a warning with a text showing the 2 values and also log the 
+                    //value of the last captured Continuation.
+                    else
+                    {
+                        logger.LogError("The prepopulated documents and the new documents don't match.  Preconfigured Docs =" + config.PreCreatedDocuments + ", New Documents = "+ documentTotal);
```

Comment:
Can be simplified with `$`:

```suggestion
                        logger.LogError($"The prepopulated documents and the new documents don't match.  Preconfigured Docs = {config.PreCreatedDocuments}, New Documents {documentTotal}");
```